.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -1,49 +1,46 @@
+## Purpose of this PR
 
-### Contribution checklist (MS Employees Only):
-- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
-- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
-- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)
+What's the purpose of this PR? Check all that apply. This is **mandatory**!
 
-If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).
+  - [ ] new service onboarding
+  - [ ] new API version
+  - [ ] update existing version for new feature
+  - [ ] update existing version to fix swagger quality issue in s360
+  - [ ] other, please clarify:
+    - _edit this with your clarification_
 
-### ARM API Review Checklist
+## Due diligence checklist
 
-> **Applicability**: :warning:
->
-> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
-> - Change to data plane APIs
-> - Adding new properties
-> - All removals
+To merge this PR, you **must** go through the following checklist and confirm you understood 
+and followed the instructions by checking all the boxes:
 
-Otherwise your PR may be subject to ARM review requirements. Complete the following:
-- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
-  - Adding a new service
-  - Adding new API(s)
-  - Adding a new API version
-    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Note that this doesn't apply if you are trying to merge a PR that was previously in the private repository.
+- [ ] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data-plane related specifications.
+- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy).
+- [ ] If I am adding a new service, API, or API version, I have ensured to copy the existing version into a new
+  directory structure for first commit and then I pushed new changes, including version updates, in separate commits.  
```

Comment:
I believe this rule might not be getting followed properly when the open api hub is used to generate the PR. The version string is possibly being set to the new version as a part of the first commit itself when that tool is used. This needs to be verified.

Reply:
I remove this whole section:

> - [ ] If I am adding a new service, API, or API version, I have ensured to copy the existing version into a new

But I took this part:

> [OpenAPI Hub](https://aka.ms/openapihub) to initialize the PR for adding a new version

Adapted it, and move it to the "purpose of this PR" checklist.